Game/ConsoleApplication2/GameObject.h

```diff
@@ -1,13 +1,26 @@
 #pragma once
 #include <SFML\Graphics.hpp>
+#include "SimpleTypes.h"
+#include "EventListener.h"
 
-class GameObject
+class GameObject : public EventListener
```

Comment:
I would argue that not every game object needs to be an event listener, but its not a massive issue